shippable.yml

```diff
@@ -0,0 +1,62 @@
+
+language: c
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  matrix:
+    - CONF=""
+    # - CONF="--disable-abi-compat"
+    # - CONF="--enable-schedule-sp"
+    # - CONF="--enable-schedule-iquery"
+    # - CONF="--enable-dpdk-zero-copy"
+    - CROSS_ARCH="arm64"
+    # - CROSS_ARCH="armhf" CFLAGS="-march=armv7-a"
+    # - CROSS_ARCH="powerpc"
+    # - CROSS_ARCH="i386"
+  global:
+    - PLACEHOLDER=""
+
+build:
+  pre_ci:
+    # use Dockerfile to install additional CI dependencies
+    - docker build -t=odp/dev ./scripts
+
+  # use image built in 'pre_ci' for CI job 
+  pre_ci_boot:
+    image_name: odp/dev
+    image_tag: latest
+    pull: false
+    options: 
+
+  ci:
+    - echo 1000 | sudo tee /proc/sys/vm/nr_hugepages
+    - sudo mkdir -p /mnt/huge
+    - sudo mount -t hugetlbfs nodev /mnt/huge
+    - mkdir -p /dev/shm/odp
+    - export CUNIT_VERSION=2.1-3
```

Comment:
This should not be necessary. Ubuntu 16.04 already provides CUnit 2.1-3. So it is better to use system one.